linux/net/rina/common.c

```diff
@@ -39,7 +39,7 @@ EXPORT_SYMBOL(is_port_id_ok);
 #define CEP_ID_WRONG -1
 
 bool is_cep_id_ok(cep_id_t id)
-{ return true; /* FIXME: Bummer, add it */ }
+{ return id >= 0 ? true : false; }
```

Comment:
Before the method was not really implemented (as you see always returned true), now it checks that the cep-id value is >= 0  (valid cep-id), otherwise returns false (negative values)